src/js/components/RangeInput/doc.js

```diff
@@ -47,7 +47,9 @@ export const themeDoc = {
   },
   'rangeInput.extend': {
     description: 'Any additional style for the RangeInput.',
-    type: 'string | (props) => {}',
+    type: `string | (props) => \`
+      color: #c9c19f;
```

Comment:
```suggestion
      color: #44444;
```
Choosing a color that is less distracting.

Reply:
Or another idea is 
```suggestion
      any CSS styling;
```
so users won't be confused it is only for colors, thoughts?

Reply:
Actually, the second one looks better. I'm going to have a little time on holiday monday, so I'll be sure to do that. 

Thanks!